addons/website_slides/static/src/js/slides_course_quiz.js

```diff
@@ -114,13 +165,22 @@ odoo.define('website_slides.quiz', function (require) {
             }).then(function (quiz_data) {
                 self.quiz = {
                     questions: quiz_data.slide_questions || [],
+                    questionsCount: quiz_data.slide_questions.length || 0,
                     quizAttemptsCount: quiz_data.quiz_attempts_count || 0,
                     quizKarmaGain: quiz_data.quiz_karma_gain || 0,
                     quizKarmaWon: quiz_data.quiz_karma_won || 0,
                 };
             });
         },
 
+        _renderQuestions: function () {
+            this.$('.o_wslides_js_lesson_quiz_question').each(function () {
+                var $question = $(this);
+                $question.find('.o_wslides_js_quiz_sequence_handler').addClass('d-none');
+                $question.find('.o_wslides_js_quiz_edit_del').addClass('d-none');
+            });
```

Comment:
You should be able to replace this whole block with:

```
this.$('.o_wslides_js_lesson_quiz_question .o_wslides_js_quiz_sequence_handler', '.o_wslides_js_lesson_quiz_question .o_wslides_js_quiz_edit_del')
    .addClass('d-none');
```

Reply:
Also, this method doesn't actually `render` anything, it just hides stuff, so the naming is not adequate.